### ISSUE TITLE
[Search] TypeError: unhashable type: 'slice'

### ISSUE THREAD

**Issue by cool-aid-man** [kaogurai/cogs#27]
Hey ryan, 
Getting this on `google` command
```py
Traceback (most recent call last):
  File "{HOME}/redenv/lib/python3.11/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "{HOME}/.local/share/Red-DiscordBot/data/pp/cogs/CogManager/cogs/search/search.py", line 122, in websearch
    for result in results[i : i + 3]:
                  ~~~~~~~^^^^^^^^^^^
TypeError: unhashable type: 'slice'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "{HOME}/redenv/lib/python3.11/site-packages/discord/ext/commands/bot.py", line 1350, in invoke
    await ctx.command.invoke(ctx)
  File "{HOME}/redenv/lib/python3.11/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "{HOME}/redenv/lib/python3.11/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: unhashable type: 'slice'

```
> - For example it will error on - `<p>Google dapletkoma` or `<p>google dhjkajddjkhajkda `

**Comment by Ryan5453** [comment]
Fixed in v2.0.2

**Comment by cool-aid-man** [comment]
Thanks Ryan
